backend/inc/inc.mod_config.php

```diff
@@ -80,6 +80,7 @@
 mip_forms_ob_start();
 eval(' ?>' . $default);
 $default_arr = mip_forms_get_array();
+//$cms_mod = empty($cms_mod) ?  : $cms_mod;
```

Comment:
Can be removed

Reply:
I have removed it. Comes with the next update.